.github/workflows/build.yaml

```diff
@@ -6,17 +6,21 @@ on:
       refLsp4ij:
         description: 'Reference/branch for Lsp4ij checkout'
         type: string
-        required: true
+        required: false
         default: main
       lsp4ijBranch:
         description: 'PR number or branch name for Artifact upload'
         type: string
-        required: true
+        required: false
       useLocalPlugin:
         description: 'Use lsp4ij locally'
-        required: true
+        required: false
```

Comment:
The `required: true` parameter indicates that this input must be provided whenever the workflow is triggered. In this case, when the workflow is called from the cronJob.yaml file, it passes the values of `refLsp4ij`, `lsp4ijBranch`, and `useLocalPlugin`. Given that these values must be passed for the job to run, is there a specific reason for setting `required: false` here?

Reply:
@anusreelakshmi934 

The reason I changed required to `false` is because I had added some additional jobs, and within those jobs, I triggered a workflow call using `uses: ./.github/workflows/build.yaml`. However, since I have only added one new job( `call-build-workflow-slack-notification` ) that does not use `uses: ./.github/workflows/build.yaml`, we can keep required set to `true`.